### PR TITLE
Admeme Language Commands (Attempt 2)

### PR DESCRIPTION
**Pull request by Mnemotechnician**
# Description
#562 but using commands instead of tricks.

Less user-friendly but makes you look like a hacker when you use them :trollface:

<details><summary><h1>Media</h1></summary><p>

image
image
image

(ignore the missing locale, I fixes)

</p></details>

# Changelog

:cl:
- add: Admin tooling: added several admin commands to help manipulate entities' languages.

### CODE REVIEW

File: Content.Server/Language/Commands/AdminLanguageCommand.cs [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    private LanguageSystem Languages => _languagesField ??= GetSys<LanguageSystem>();
+
+    [CommandImplementation("add")]
+    public EntityUid AddLanguage(
+        [CommandInvocationContext] IInvocationContext ctx,
+        [PipedArgument] EntityUid input,
+        [CommandArgument] ValueRef<string, Prototype<LanguagePrototype>> @ref,
+        [CommandArgument] bool canSpeak = true,
+        [CommandArgument] bool canUnderstand = true
+        )
```
Comment (DEATHB4DEFEAT) [inline Content.Server/Language/Commands/AdminLanguageCommand.cs:25]:
```suggestion
    )
```

File: Content.Server/Language/Commands/AdminLanguageCommand.cs [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    [CommandImplementation("rm")]
+    public EntityUid RemoveLanguage(
+        [CommandInvocationContext] IInvocationContext ctx,
+        [PipedArgument] EntityUid input,
+        [CommandArgument] ValueRef<string, Prototype<LanguagePrototype>> @ref,
+        [CommandArgument] bool removeSpeak = true,
+        [CommandArgument] bool removeUnderstand = true
+        )
```
Comment (DEATHB4DEFEAT) [inline Content.Server/Language/Commands/AdminLanguageCommand.cs:50]:
```suggestion
    )
```

File: Content.Server/Language/Commands/AdminLanguageCommand.cs [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    [CommandImplementation("add")]
+    public EntityUid AddLanguage(
+        [CommandInvocationContext] IInvocationContext ctx,
+        [PipedArgument] EntityUid input,
+        [CommandArgument] ValueRef<string, Prototype<LanguagePrototype>> @ref,
+        [CommandArgument] bool canSpeak = true,
+        [CommandArgument] bool canUnderstand = true
+        )
+    {
+        var language = @ref.Evaluate(ctx)!;
```
Comment (DEATHB4DEFEAT) [inline Content.Server/Language/Commands/AdminLanguageCommand.cs:27]: What is this?

File: Content.Server/Language/Commands/AdminTranslatorCommand.cs
Comment (DEATHB4DEFEAT) [inline Content.Server/Language/Commands/AdminTranslatorCommand.cs:1]: Localize the errors

File: Resources/Locale/en-US/language/commands.ftl
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/language/commands.ftl:1]: I remember there being a separate part for the examples, but that's probably just pre-toolshed.

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

File: Content.Server/Language/Commands/AdminLanguageCommand.cs (hunk 3 above)
Comment (Mnemotechnician) [reply Content.Server/Language/Commands/AdminLanguageCommand.cs:27]: I just did it the same way as it was done in tag command toolshed.

**Comment by DangerRevolution** [comment]
what if you add tricks AND commands

**Review by DangerRevolution — APPROVED** [review]

**Comment by DangerRevolution** [comment]
This is stupid, should've been tricks but works as is >:(

**Comment by DangerRevolution** [comment]
Mind you, I haven't tested it.

**Comment by Mnemotechnician** [comment]
This needs review

**Review by VMSolidus — APPROVED** [review]

**Pull request merged**